### DISCUSSION TITLE
窗口最前

### DISCUSSION THREAD

**Discussion by zszfympx** [amll-dev/applemusic-like-lyrics#245] [Ideas]
在AMLL Player中能否添加一个特定的选项，使得tauri的窗口总在最前，我记得tauri是有这个的吧（）

**Comment by zszfympx** [comment]
https://tauri.app/v1/api/js/window/

**Comment by Steve-xmh** [comment]
是可以，但是作者用 PowerToys 直接 Ctrl+Win+T 就可以置顶任意一个窗口了所以就没多管（
到后面确有需求了再倒腾吧（
